types/react-imgpro/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please don't disable rules unless you really have to. Most of them are there for a good reason. You can have per-line disables if needed, but they should be justified. Most packages do not disable any rules.